src/action/ActionImpl.ts

```diff
@@ -8,6 +8,14 @@ interface ActionImplOptions {
   history?: History;
 }
 
+/**
+ * Extends the configured keywords to include the section
+ * This allows section names to be searched for.
+ */
+const extendKeywords = ({ keywords = "", section = "" }: Action): string => {
+  return `${keywords} ${section}`;
```

Comment:
For the sake of not having that extra space if there is no section: 

```suggestion
  return `${keywords} ${section}`.trim();
```